Sources/SwiftLexicalLookup/LookupName.swift

```diff
@@ -139,6 +139,19 @@ import SwiftSyntax
   case implicit(ImplicitDecl)
   /// Dollar identifier introduced by a closure without parameters.
   case dollarIdentifier(ClosureExprSyntax, strRepresentation: String)
+  /// Represents equivalent identifiers grouped together.
+  /// - Important: The array should be non-empty.
+  ///
+  /// ### Example:
+  /// ```swift
+  /// switch X {
+  /// case .a(let x), .b(let x):
+  ///   print(x) // <-- lookup here
+  /// }
+  /// ```
+  /// For lookup at the given position, the result
+  /// contains only one (composite) name
+  case compositeName([LookupName])
```

Comment:
I don’t like to use `compositeName` here because we also call function names with argument labels (like `foo(a:b:)`) compound names, which is very close to composite name. How about something that like `multipleDeclarations` or something like that?

Reply:
`equivalentNames` sounds good to me. 👍🏽